src/solvers/petscdmlibmeshimpl.C

```diff
@@ -907,7 +907,9 @@ static PetscErrorCode  DMSetUp_libMesh(DM dm)
     ierr = DMSNESSetFunction(dm, SNESFunction_DMlibMesh, (void *)dm); CHKERRQ(ierr);
     ierr = DMSNESSetJacobian(dm, SNESJacobian_DMlibMesh, (void *)dm); CHKERRQ(ierr);
     if (dlm->sys->nonlinear_solver->bounds || dlm->sys->nonlinear_solver->bounds_object)
-      ierr = DMSetVariableBounds(dm, DMVariableBounds_libMesh); CHKERRQ(ierr);
+      {
+        ierr = DMSetVariableBounds(dm, DMVariableBounds_libMesh); CHKERRQ(ierr);
```

Comment:
This looks like a real issue that was caught, nice!

Reply:
I would also like to cherry-pick this. I've been sitting on the latest 1.7.0 RC for a while, but there haven't been many fixes recently... probably should just go ahead and pull the trigger.